fsm.go

```diff
@@ -268,10 +262,13 @@ func (s *stateMachine) processPacketType(p packet) []packet {
 	return nil
 }
 
-// suspicionTimeout returns the number of periods to wait before declaring a
-// suspect failed.
-func (s *stateMachine) suspicionTimeout() int {
-	return int(3*math.Log(float64(len(s.members)))) + 1
+// logn2 returns 2*log(n) rounded up, where n is the size of the network.
+// Each message must be sent a small multiple of log(n) times to ensure
+// reliable dissemination. Consequently, this is also the dissemination
+// timescale, and by extension the number of profile periods to wait before
+// declaring a suspect failed.
+func (s *stateMachine) logn2() int {
```

Comment:
Only suggestion is to call this something like `disseminationFactor` or whatever in case the number ever changes again, so there's no need to rename the function.